Gopkg.toml

```diff
@@ -106,3 +106,7 @@ required = [
 [[override]]
   name = "k8s.io/gengo"
   revision = "b90029ef6cd877cb3f422d75b3a07707e3aac6b7"
+
+[[constraint]]
+  name = "github.com/openshift/client-go"
+  version = "3.9.0"
```

Comment:
Any reason for version 3.9.0? I feel like we should maybe be using `release-4.2` at least

Reply:
Actually `release-4.2` is probably changing regularly. Maybe the most recent commit from `release-4.2`.

Reply:
https://github.com/openshift/cluster-authentication-operator/blob/master/glide.yaml#L20-L21

Reply:
https://github.com/openshift/cluster-image-registry-operator/blob/master/Gopkg.toml#L45-L47

Reply:
Nope, no reason. I just ran `dep ensure -add ...` which added that version. I'll update w/ the override tag.